packages/frontend/pages/index.tsx

```diff
@@ -3,16 +3,15 @@ import { useTranslation } from 'next-i18next';
 import { Button, Container } from '@primitives';
 import { Http } from '@vse-bude/shared';
 import { Layout } from '@components';
-import { CookieStorage } from '@helpers';
+import { auth } from '@helpers';
 import { useAppDispatch, useTypedSelector } from '@hooks';
 import { shallowEqual } from 'react-redux';
 import { fetchRandomData, fetchRandomDataSSR, wrapper } from 'store';
 
 export const getServerSideProps = wrapper.getServerSideProps(
   (store) => async (ctx) => {
-    const storage = new CookieStorage(ctx);
     const { locale } = ctx;
-    const httpClient = new Http(process.env.NEXT_PUBLIC_API_ROUTE, storage);
+    const httpClient = new Http(process.env.NEXT_PUBLIC_API_ROUTE, auth);
```

Comment:
will we have a access token in auth?

Reply:
no, i didn't think about it. I will do it